.github/workflows/changelog.yaml

```diff
@@ -0,0 +1,20 @@
+name: '📝 Create/Update Release Draft & Release Bump PR'
+
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - build.yaml
+  workflow_dispatch:
+  repository_dispatch:
+    types:
+      - update-prep-command
+
+jobs:
+  call:
+    uses: jellyfin/jellyfin-meta-plugins/.github/workflows/changelog.yaml@master
+    with:
+      repository-name: jellyfin/jellyfin-plugin-template
+    secrets:
+      token: ${{ secrets.JF_BOT_TOKEN }}
```

Comment:
```suggestion
      token: ${{ secrets.GITHUB_TOKEN }}
```
Since this is the public template for our plugins we should not use a token in the workflows that is only accessable to us :wink: 
(the default `GITHUB_TOKEN` is enough for all workfows, we only use our `BOT` to have a fancy committer/release creator)